examples/demo-plugin/driver.js

```diff
@@ -1,7 +1,10 @@
-const DeviceDriverBase = require('detox/src/devices/drivers/DeviceDriverBase');
+const DeviceDriverBase = require('detox/src/devices/runtime/drivers/DeviceDriverBase');
```

Comment:
Just so you're aware, it is a bad practice to allow plugins to go deep inside the module: either Detox should export some endpoint like `require('detox/plugin-api')`, or export that stuff at the top level `const { plugin } = require('detox')`.